test/UnitTests/MSTest.Analyzers.UnitTests/ClassCleanupShouldBeValidAnalyzerTests.cs

```diff
@@ -627,4 +627,39 @@ public abstract class MyTestClass
 
         await VerifyCS.VerifyAnalyzerAsync(code);
     }
+
+    public async Task WhenClassCleanupIsOnSealedClassNotMarkedWithTestClass_AndWithInheritanceBehavior_Diagnostic()
+    {
+        string code = """
+            using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+            public sealed class MyTestClass
+            {
+                [ClassCleanup(InheritanceBehavior.BeforeEachDerivedClass)]
```

Comment:
Would we report if the `InheritanceBehavior` isn't set?

Reply:
yes and we have test for it

Reply:
It seems that this test is not bringing much then. You cannot tell if you are reporting because of your change or because we always report if class is not `[TestClass]`.